doc/sphinx/addendum/program.rst

```diff
@@ -295,6 +295,14 @@ optional tactic is replaced by the default one if not specified.
 
    Start the proof of the next unsolved obligation.
 
+.. cmd:: Final Obligation {? with @ltac_expr }
```

Comment:
This omits the `ident` from `Final Obligation of @ident OPT ( with @ltac_expr )`. 
 You need the change I requested in https://github.com/coq/coq/pull/16817#discussion_r1024376502 to be accurate here.